commonslibrary/settings/base.py

```diff
@@ -282,12 +282,15 @@
 # yapf: disable
 CKEDITOR_CONFIGS = {
     'default': {
-        'skin': 'moono-lisa',
```

Comment:
I think moona-lisa fits in with the design a lot better, thoughts @pauledevans?

Reply:
@pauledevans